components/dialog/test/dialog.vdiff.js

```diff
@@ -100,6 +125,23 @@ describe('dialog', () => {
 				await expect(document).to.be.golden();
 			});
 
+			it('delayed-content', async() => {
+				const el = await fixture(
+					`<d2l-dialog title-text="Delayed Dialog"><${delayedTag}></${delayedTag}></d2l-dialog>`,
+					{
+						awaitLoadingComplete: false
```

Comment:
This is a special configuration option because otherwise the `fixture` call itself would wait for all the descendants to be loading-complete. In this case, we want to bypass that since we want to make sure that dialog itself waits for them.